README.md

```diff
@@ -4,21 +4,108 @@
 ## What is ocdev?
 OpenShift Command line for Developers
 
+## Pre-requisites
+- OpenShift version 3.7 and up
+
+To use ocdev you need access to OpenShift instance and OpenShift CLI installed on your local machine (`oc` should be in your $PATH).
+
+
+### OpenShift instance
+You can use [Minishift](https://docs.openshift.org/latest/minishift/index.html) to get a local instance of OpenShift. However ocdev can be used with any instance of OpenShift.
+
+### OpenShift CLI 
+There are different ways to install OpenShift CLI. 
+Please follow [OpenShift documentation](https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli).
+
+
 ## Installation
-You can [download](https://dl.bintray.com/ocdev/ocdev/latest/) latest binaries (build automatically from current master branch).
+You can download latest master builds from [Bintray](https://dl.bintray.com/ocdev/ocdev/latest/) or 
+builds for released versions from [GitHub releases page](https://github.com/kadel/ocdev/releases).
+
+### MacOS
+1. First you need enable `kadel/ocdev` Homebrew Tap:
+    ```sh
+    brew tap kadel/ocdev
+    ```
```

Comment:
This also concerns me. The issue is once you mention a location users will start using it and would expect it working every time. So better to have proper brewtab for per-releases i.e. alpha beta etc or do not have it all together for now.

Reply:
> So better to have proper brewtab for per-releases i.e. alpha beta etc or do not have it all together for now.

don't understand this one :-(

Reply:
I mean https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask 

We have a cask for alpha, beta releases.

Reply:
I'm still not sure if I understand it :-( 

But Cask is only for MacOS gui applications or sw distributed as binaries no?

If you are suggesting creating different taps for alpha/beta/stable.I'm not sure if we need it. I would rather have one channel for released versions and other for master builds.

I don't expect that we will keep releasing beta or alpha releases after first stable release.
Master builds will act as alpha/beta releases and those have a separate channel.

Reply:
@kadel cask is for any applications on macOS. You can define taps for alpha/beta releases, another for master build and another for nightly. IMO that is more than enough. For Minishift we have only beta and GA channel.